frontend/src/components/Pages/Dashboard/Dashboard.jsx

```
@@ -276,146 +209,103 @@ const Dashboard = () => {
   return (
-    <div className="flex bg-gray-900 min-h-screen">
-      <DashboardSideNavbar 
-        isOpen={sidebarOpen} 
-        setIsOpen={setSidebarOpen}
-      />
-      <div className="flex-1 p-4 sm:p-6 md:p-8 ml-0 md:ml-20 lg:ml-64 transition-all duration-300">
+    <div className="flex bg-gray-900 min-h-screen text-white">
+      <DashboardSideNavbar isOpen={sidebarOpen} setIsOpen={setSidebarOpen} />
+      <main className={`flex-1 p-4 sm:p-6 md:p-8 ${sidebarOpen ? 'ml-64' : 'ml-0 md:ml-20'} lg:ml-64 transition-all duration-300 ease-in-out overflow-y-auto`}>
+         <button onClick={() => setSidebarOpen(!sidebarOpen)} className="md:hidden fixed top-4 left-4 z-50 p-2 bg-gray-700 rounded text-white hover:bg-gray-600 focus:outline-none focus:ring-2 focus:ring-gray-500" aria-label="Toggle sidebar">
+            <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" strokeWidth={1.5} stroke="currentColor" className="w-6 h-6"><path strokeLinecap="round" strokeLinejoin="round" d="M3.75 6.75h16.5M3.75 12h16.5m-16.5 5.25h16.5" /></svg>
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)